### ISSUE TITLE
Error while running `babelish csv2android` because of thor dependency

### ISSUE THREAD

**Issue by 4brunu** [netbe/Babelish#141]
Hi,

I have an error while running `bundle exec babelish csv2android` because of thor dependency.

```
bundle exec babelish csv2android
bundler: failed to load command: babelish (/usr/local/lib/ruby/gems/2.7.0/bin/babelish)
NoMethodError: private method `options' called for #<Commandline:0x00007ff78127b908>
Did you mean?  options=
  /usr/local/lib/ruby/gems/2.7.0/gems/thor-1.1.0/lib/thor/shell/basic.rb:393:in `quiet?'
  /usr/local/lib/ruby/gems/2.7.0/gems/thor-1.1.0/lib/thor/shell/basic.rb:97:in `say'
  /usr/local/lib/ruby/gems/2.7.0/gems/thor-1.1.0/lib/thor/shell.rb:60:in `say'
  /usr/local/lib/ruby/gems/2.7.0/gems/babelish-0.6.5/lib/babelish/commandline.rb:158:in `block in csv2base'
  /usr/local/lib/ruby/gems/2.7.0/gems/babelish-0.6.5/lib/babelish/commandline.rb:150:in `each'
  /usr/local/lib/ruby/gems/2.7.0/gems/babelish-0.6.5/lib/babelish/commandline.rb:150:in `each_with_index'
  /usr/local/lib/ruby/gems/2.7.0/gems/babelish-0.6.5/lib/babelish/commandline.rb:150:in `csv2base'
  /usr/local/lib/ruby/gems/2.7.0/gems/babelish-0.6.5/lib/babelish/commandline.rb:38:in `block (2 levels) in <class:Commandline>'
  /usr/local/lib/ruby/gems/2.7.0/gems/thor-1.1.0/lib/thor/command.rb:27:in `run'
  /usr/local/lib/ruby/gems/2.7.0/gems/thor-1.1.0/lib/thor/invocation.rb:127:in `invoke_command'
  /usr/local/lib/ruby/gems/2.7.0/gems/thor-1.1.0/lib/thor.rb:392:in `dispatch'
  /usr/local/lib/ruby/gems/2.7.0/gems/thor-1.1.0/lib/thor/base.rb:485:in `start'
  /usr/local/lib/ruby/gems/2.7.0/gems/babelish-0.6.5/bin/babelish:6:in `<top (required)>'
  /usr/local/lib/ruby/gems/2.7.0/bin/babelish:23:in `load'
  /usr/local/lib/ruby/gems/2.7.0/bin/babelish:23:in `<top (required)>'
```

Here is my Gemfile
```
source "https://rubygems.org"

gem 'babelish', '0.6.5'
```

Here is my Gemfile.lock
```
GEM
  remote: https://rubygems.org/
  specs:
    activesupport (6.1.2)
      concurrent-ruby (~> 1.0, >= 1.0.2)
      i18n (>= 1.6, < 2)
      minitest (>= 5.1)
      tzinfo (~> 2.0)
      zeitwerk (~> 2.3)
    addressable (2.7.0)
      public_suffix (>= 2.0.2, < 5.0)
    babelish (0.6.5)
      google_drive (~> 3.0.5)
      highline
      json
      nokogiri
      rack (>= 1.6.11)
      thor
    concurrent-ruby (1.1.8)
    declarative (0.0.20)
    declarative-option (0.1.0)
    faraday (1.3.0)
      faraday-net_http (~> 1.0)
      multipart-post (>= 1.2, < 3)
      ruby2_keywords
    faraday-net_http (1.0.1)
    gems (1.2.0)
    google-api-client (0.53.0)
      google-apis-core (~> 0.1)
      google-apis-generator (~> 0.1)
    google-apis-core (0.2.1)
      addressable (~> 2.5, >= 2.5.1)
      googleauth (~> 0.14)
      httpclient (>= 2.8.1, < 3.0)
      mini_mime (~> 1.0)
      representable (~> 3.0)
      retriable (>= 2.0, < 4.0)
      rexml
      signet (~> 0.14)
      webrick
    google-apis-discovery_v1 (0.1.0)
      google-apis-core (~> 0.1)
    google-apis-generator (0.1.2)
      activesupport (>= 5.0)
      gems (~> 1.2)
      google-apis-core (~> 0.1)
      google-apis-discovery_v1 (~> 0.0)
      thor (>= 0.20, < 2.a)
    google_drive (3.0.6)
      google-api-client (>= 0.11.0, < 1.0.0)
      googleauth (>= 0.5.0, < 1.0.0)
      nokogiri (>= 1.5.3, < 2.0.0)
    googleauth (0.15.1)
      faraday (>= 0.17.3, < 2.0)
      jwt (>= 1.4, < 3.0)
      memoist (~> 0.16)
      multi_json (~> 1.11)
      os (>= 0.9, < 2.0)
      signet (~> 0.14)
    highline (2.0.3)
    httpclient (2.8.3)
    i18n (1.8.8)
      concurrent-ruby (~> 1.0)
    json (2.5.1)
    jwt (2.2.2)
    memoist (0.16.2)
    mini_mime (1.0.2)
    mini_portile2 (2.5.0)
    minitest (5.14.3)
    multi_json (1.15.0)
    multipart-post (2.1.1)
    nokogiri (1.11.1)
      mini_portile2 (~> 2.5.0)
      racc (~> 1.4)
    os (1.1.1)
    public_suffix (4.0.6)
    racc (1.5.2)
    rack (2.2.3)
    representable (3.0.4)
      declarative (< 0.1.0)
      declarative-option (< 0.2.0)
      uber (< 0.2.0)
    retriable (3.1.2)
    rexml (3.2.4)
    ruby2_keywords (0.0.4)
    signet (0.14.1)
      addressable (~> 2.3)
      faraday (>= 0.17.3, < 2.0)
      jwt (>= 1.5, < 3.0)
      multi_json (~> 1.10)
    thor (1.1.0)
    tzinfo (2.0.4)
      concurrent-ruby (~> 1.0)
    uber (0.1.0)
    webrick (1.7.0)
    zeitwerk (2.4.2)

PLATFORMS
  ruby

DEPENDENCIES
  babelish (= 0.6.5)

BUNDLED WITH
   2.1.4
```

If I downgrade the dependency `thor` to version 1.0.1 it works.

**Comment by borut-t** [comment]
I have the same issue. Thanks @4brunu 🙌

**Comment by borut-t** [comment]
Btw, until this gets merged you can add `gem 'thor', '1.0.1'` to the Gemfile.

**Comment by 4brunu** [comment]
Thanks for the tip 👍
